packages/embed/package.json

```diff
@@ -29,7 +28,7 @@
     "prestart": "npm run build:babel",
     "start": "concurrently --kill-others --names \"babel,server,webpack\" \"npm run start:babel\" \"npm run start:server\" \"npm run start:webpack\"",
     "start:babel": "npm run build:babel -- --skip-initial-build --watch",
-    "start:server": "node-dev --no-notify --respawn hostDevServer",
+    "start:server": "node-dev --no-notify --respawn hostDevServer.mjs",
```

Comment:
Need to ensure node requirements for modules are met

Reply:
Node.js supports `.mjs` since 12 or better at 14, which are already past LTS. Do you think we still should add `{ engine: { node: '>=16' } }`?

Reply:
While the introduced supports of `mjs` is past LTS, I'm thinking proper interop between mjs and cjs was introduced only in the latest version.

I checked deps, and I think we're good here. Probably worth to add the `{ engine: { node: '>=16' } }` property, but given we support anything NodeJS community supports, we're good either.